tasks/main.yml

```diff
@@ -45,7 +45,9 @@
   package:
     name: amazon-cloudwatch-agent
     state: present
-  ignore_errors: yes
+  failed_when: 
+    - not (already_inst.rc == 0)
+    - not (already_inst.rc == 126)
```

Comment:
This was failing normally which makes it harder to tell if the role worked properly. If this task actually failed it would have caused problems.

Now, if `amazon-cloudwatch-agent` is installed this will succeed with `rc` of 0. If it's not installed then it will now succeed but with `rc` of 126.

Reply:
Good change although I have never been happy with this